src/PythonMigrationViewExtension/PythonMigrationViewExtension.cs

```diff
@@ -21,6 +20,8 @@ internal class PythonMigrationViewExtension : IViewExtension
         private NotificationMessage IronPythonNotification { get; set; }
         internal WorkspaceModel CurrentWorkspace { get; set; }
         internal GraphPythonDependencies PythonDependencies { get; set; }
+        internal Dispatcher Dispatcher { get; set; }
```

Comment:
should probably make both of these `private` and use methods to delegate stuff to them if needed from outside this extension.